scripts/build-dist.js

```diff
@@ -45,6 +45,8 @@ babelConfig.babelrc = false;
 babelConfig.presets = babelConfig.presets.map((preset) => {
     return preset === 'es2015' ? 'es2015-rollup' : preset;
 });
+babelConfig.plugins = babelConfig.plugins || [];
+babelConfig.plugins.push(['transform-react-remove-prop-types', {'mode': 'wrap'}]);
```

Comment:
I think a better approach would be to add this to the `.babelrc` when `"env"` is `"production"`.